composer.json

```diff
@@ -31,7 +31,8 @@
   "minimum-stability": "dev",
   "prefer-stable":     true,
   "require":     {
-    "dreamfactory/df-core": "~0.22"
+    "php": "^8.0",
```

Comment:
@daniilly @alexanderko Title says PHP 8.1 but here you're requiring 8.0, why is that?

Reply:
Since there are no usage of php 8.1 specific functions, it is decided to use 8.0 as base and move to new minor version in the future when needed